index.d.ts

```diff
@@ -1,4 +1,10 @@
-export interface PromptOptions {
+export interface QuestionOptions {
+  stdin?: NodeJS.ReadStream & {
+    fd: 0;
+  };
+  stdout?: NodeJS. WriteStream & {
```

Comment:
```suggestion
  stdout?: NodeJS.WriteStream & {
```

Reply:
Also, using `NodeJS` namespace need `@types/node` to be installed (`npm i -D @types/node`)

Reply:
thanks for reply! 
I fixed whitespace typo, and added @types/node as dev dependency.